### PR TITLE
Add show sweeps command

### PR DESCRIPTION
**Pull request by manskx**
# Before submitting

## What does this PR do?

This PR adds support for viewing the sweeps.

Screenshot 2022-09-01 at 18 58 54

Screenshot 2022-09-01 at 18 59 15

Fixes https://github.com/Lightning-AI/lightning-hpo/issues/50

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: lightning_hpo/controllers/sweeper.py [hunk 1]
```diff
@@ -75,6 +76,9 @@ def run_sweep(self, config: SweepConfig) -> str:
         #     return f"Updated code for Sweep {config.sweep_id}."
         return f"The current Sweep {config.sweep_id} is running. It couldn't be updated."
 
+    def get_db_url(self) -> str:
```
Comment (tchaton) [inline lightning_hpo/controllers/sweeper.py:79]: Let's follow a command name convention: `show_sweeps_handler`

File: lightning_hpo/commands/sweep/list_sweeps.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        pass
+
+
+class SweepsList(Formatable):
+    def __init__(self, sweeps: List[dict]):
+        self.sweeps = sweeps
+
+    def as_table(self) -> Table:
+        # TODO: Improve the table format.
+        table = Table(
```
Comment (tchaton) [inline lightning_hpo/commands/sweep/list_sweeps.py:23]: Let's use a Tree format with the Trial information.

File: lightning_hpo/commands/sweep/list_sweeps.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                sweep["framework"],
+                sweep["cloud_compute"],
+                str(sweep["n_trials"]),
+            )
+        return table
+
+
+class ShowSweepsListCommand(ClientCommand):
+    def run(self) -> None:
+        # 1: Get the database URL
```
Comment (tchaton) [inline lightning_hpo/commands/sweep/list_sweeps.py:40]: Let's add an argument parser to filter the sweep, trials.

File: lightning_hpo/commands/sweep/list_sweeps.py (hunk 2 above)
Comment (manskx) [reply lightning_hpo/commands/sweep/list_sweeps.py:23]: There's a lot of information, what information do you want to display ? e.g. this is the json of one sweep with 3 trials 

```json

{
   "requirements":[
      
   ],
   "script_args":[
      
   ],
   "trials":{
      "0":{
         "best_model_score":0.07500000298023224,
         "monitor":"val_acc",
         "best_model_path":"None",
         "status":"succeeded",
         "params":{
            "params":{
               "model.lr":0.08025794913236321
            }
         },
         "exception":"None"
      },
      "1":{
         "best_model_score":0.09375,
         "monitor":"val_acc",
         "best_model_path":"None",
         "status":"succeeded",
         "params":{
            "params":{
               "model.lr":0.099850192637455
            }
         },
         "exception":"None"
      },
      "2":{
         "best_model_score":0.125,
         "monitor":"val_acc",
         "best_model_path":"None",
         "status":"succeeded",
         "params":{
            "params":{
               "model.lr":0.033720632727005546
            }
         },
         "exception":"None"
      }
   },
   "sweep_id":"mansy-942286bf",
   "n_trials":3,
   "trials_done":3,
   "framework":"pytorch_lightning",
   "num_nodes":1,
   "direction":"maximize",
   "status":"succeeded",
   "id":1,
   "distributions":{
      "model.lr":{
         "distribution":"uniform",
         "params":{
            "params":{
               "low":0.001,
               "high":0.1
            }
         }
      }
   },
   "script_path":"train.py",
   "simultaneous_trials":1,
   "url":"None",
   "cloud_compute":"cpu",
   "logger":"wandb"
}
```

**Pull request merged**